Fixed typos and improved uniformity of RNW CLI command help text (#13386)
microsoft/react-native-windows: packages/@react-native-windows/cli/README.md

**Note:** Autolinking runs automatically as part of running the `run-windows` CLI command, unless the `--no-autolink` argument is used.

### Usage
Runs Windows-specific autolinking for your RNW project.
  
```bat
npx react-native autolink-windows
```
### Options

Here are the options that `react-native autolink-windows` takes:
| Option                | Input Type | Description                                      |
|-----------------------|------------|--------------------------------------------------|
| `--logging`           | boolean    | Verbose output logging                           |
| `--check`             | boolean    | Only check whether any autolinked files need to change |
| `--sln`               | string     | Override the app solution file determined by 'react-native config', i.e. `windows\myApp.sln` |
| `--proj`              | string     | Override the app project file determined by 'react-native config', i.e. `windows\myApp\myApp.vcxproj` |
| `--no-telemetry`      | boolean    | Disables sending telemetry that allows analysis of usage and failures of the react-native-windows CLI |

> | `-h`, `--help`        | boolean    | display help for command                         |

Reviewer: ```suggestion
| `-h`, `--help`        | boolean    | Display help for command                         |
```

## `codegen-windows`

`codegen-windows` is the CLI command provided by `@react-native-windows/cli` which is used to generate some necessary Windows-specific native code for native modules.

### Usage
Runs Windows-specific codegen for native modules.
  
```bat
npx react-native codegen-windows
```
### Options

Here are the options that `react-native codegen-windows` takes:
| Option                | Input Type | Description                                      |
|-----------------------|------------|--------------------------------------------------|
| `--logging`           | boolean    | Verbose output logging                           |
| `--check`             | boolean    | Only check whether any codegen files need to change |
| `--no-telemetry`      | boolean    | Disables sending telemetry that allows analysis of usage and failures of the react-native-windows CLI |

> | `-h`, `--help`        | boolean    | display help for command                         |

Reviewer: ```suggestion
| `-h`, `--help`        | boolean    | Display help for command                         |
```

## `init-windows`

`init-windows` is the CLI command provided by `@react-native-windows/cli` which is used to initialize a new React Native for Windows project inside an existing React Native project. 

### Usage
Initializes a new RNW project from a given template.
  
```bat
npx react-native init-windows
```
### Options

Here are the options that `react-native init-windows` takes:
| Option                | Input Type | Description                                      |
|-----------------------|------------|--------------------------------------------------|
| `--logging`           | boolean    | Verbose output logging                           |
| `--template`          | string     | Specify the template to use                      |
| `--name`              | string     | The native project name. Defaults to the name property in `app.json` or `package.json` |

> | `--namespace`         | string     | The native project namespace, expressed using dots as separators, i.e. 'Level1.Level2.Level3'. Defaults to the same as name |

Reviewer: ```suggestion
| `--namespace`         | string     | The native project namespace expressed using dots as separators, i.e. 'Level1.Level2.Level3'. Defaults to the same as name |
```

> | `--overwrite`         | boolean    | Overwrite and existing files without prompting  |

Reviewer: ```suggestion
| `--overwrite`         | boolean    | Overwrite any existing files without prompting  |
```

| `--no-telemetry`      | boolean    | Disables sending telemetry that allows analysis of usage and failures of the react-native-windows CLI |

> | `-h`, `--help`        | boolean    | display help for command                         |

Reviewer: ```suggestion
| `-h`, `--help`        | boolean    | Display help for command                         |
```

## `run-windows`

`run-windows` is the CLI command provided by `@react-native-windows/cli` which is used to build and run React Native for Windows apps. 

### Usage
Builds your RNW app and starts it on a connected Windows desktop, emulator or device.
  
```bat
npx react-native run-windows
```
### Options

> **Note:** Remote Debugging was officially marked as [deprecated](https://github.com/react-native-community/discussions-and-proposals/discussions/734) in RN 0.73 and will be removed in a later release.

Here are the options that `react-native run-windows` takes:
| Option                | Input Type | Description                                      |
|-----------------------|------------|--------------------------------------------------|
| `--release`           | boolean    | Specifies a Release build                        |

> | `--root`              | string     | Override the root directory for the windows build which contains the windows folder (default: `E:\\test63`) |

Author: ```suggestion
| `--root`              | string     | Override the root directory for the project which contains the `windows` folder |
```

> | `--root`              | string     | Override the root directory for the windows build which contains the windows folder (default: `E:\\test63`) |

Reviewer: ```suggestion
| `--root`              | string     | Override the root directory for the Windows build which contains the windows folder (default: `E:\\test63`) |
```

| `--singleproc`        | boolean    | Disables multi-proc build                        |
| `--emulator`          | boolean    | Deploys the app to an emulator                   |
| `--device`            | boolean    | Deploys the app to a connected device            |
| `--target`            | string     | Deploys the app to the specified `GUID` for a device |
| `--remote-debugging`  | boolean    | **(Deprecated)** Deploys the app in remote debugging mode |
| `--logging`           | boolean    | Verbose output logging                           |
| `--no-packager`       | boolean    | Do not launch the packager while building        |
| `--bundle`            | boolean    | Enable Bundle configuration, i.e. `ReleaseBundle`/`DebugBundle` rather than `Release`/`Debug` |
| `--no-launch`         | boolean    | Do not launch the app after deployment           |
| `--no-autolink`       | boolean    | Do not run autolinking                           |
| `--no-build`          | boolean    | Do not build the solution                        |
| `--no-deploy`         | boolean    | Do not deploy the app                            |

> | `--deploy-from-layout`| boolean    | Force deploy from layout, even in release builds |

Reviewer: ```suggestion
| `--deploy-from-layout`| boolean    | Force deploy from layout even in Release builds |
```

| `--sln`               | string     | Override the app solution file determined by 'react-native config', i.e. `windows\myApp.sln` |
| `--proj`              | string     | Override the app project file determined by 'react-native config', i.e. `windows\myApp\myApp.vcxproj` |
| `--msbuildprops`      | string     | Comma separated props to pass to MSBuild, i.e. `prop1=value1,prop2=value2` |
| `--buildLogDirectory` | string     | Optional directory where MSBuild log files should be stored |
| `--info`              | boolean    | Dump environment information                     |
| `--direct-debugging`  | number     | Enable direct debugging on specified port        |
| `--no-telemetry`      | boolean    | Disables sending telemetry that allows analysis of usage and failures of the react-native-windows CLI |

> | `-h`, `--help`        | boolean    | display help for command                         |

Reviewer: ```suggestion
| `-h`, `--help`        | boolean    | Display help for command                         |
```
